crowbar_framework/app/views/nodes/_form.html.haml

```diff
@@ -6,7 +6,7 @@
 
         .btn-group.pull-right
           = link_to t("cancel"), node_path(@node.handle), :class => "btn btn-default"
-          %input.btn.btn-default{ :type => "submit", :name => "submit", :value => t(".save") }
+          %input.btn.btn-primary{ :type => "submit", :name => "submit", :value => t(".save") }
 
           - unless @node.allocated? or @node.admin?
             %input.btn.btn-default{ type: "submit", name: "submit", value: t(".allocate"), data: { confirm: t(".confirm_allocate"), title: t(".allocate_tooltip") } }
```

Comment:
(keep in mind I'm not aware of the guidelines we're using, so I may be saying something totally wrong)

Don't we want to drop this set of buttons that are at the top, since we have the same buttons at the bottom? (this was done for the proposal page in the other pull request)

Reply:
Was this comment on some older code? That was removed in a later push.